### ISSUE TITLE
[Accessibility] Keyboard navigation and buttons

### ISSUE THREAD

**Issue by Majorfi** [yearn/web-lib#9]
# What
To have the best experience for user, our lib should be accessible via keyboard navigation.
Keyboard navigation is usually correctly handled by the browsers if the correct tabs are used.
- Any element with an `onClick` event _should be_ a `<button />`
- The priority on the navigation in a group of items in the same group can be given by the `tabIndex` props, with `0` for automatic, `n` (1, 2, ..., n) for manual and `-1` to disable.
- Any element focused by keyboard navigation will have a ring around with the primary color to make it visible

# Where
- The chevron components in some banners
- The Image banner 
- Some hack on the Switch components
- Any other missing places

# Note
This _should_ be a button, but it's not mandatory. Sometime, for some reasons, it's best to keep a simple `<div />` or something else.
